lib/cancan/parameter_validators.rb

```diff
@@ -0,0 +1,7 @@
+module CanCan
+  module ParameterValidators
+    def valid_attribute_param?(attribute)
+      attribute.nil? || attribute.is_a?(Symbol) || (attribute.is_a?(Array) && attribute.first.is_a?(Symbol))
```

Comment:
do you want to check all of them? `(attribute.is_a?(Array) && attribute.all? {|a| a.is_a?(Symbol) })`

Reply:
An artifact from copying the cancan code! You are right; I should be more thorough.